src/main/java/io/cryostat/templates/TemplatesModule.java

```diff
@@ -55,4 +56,11 @@ static LocalStorageTemplateService provideLocalStorageTemplateService(
             FileSystem fs, Environment env) {
         return new LocalStorageTemplateService(fs, env);
     }
+
+    @Provides
+    @Singleton
+    static LocalProbeTemplateService provideLocalProbeTemplateService(
```

Comment:
We should probably create a new module for probes rather than piggybacking them into event templates, but for now this will do.